direct/data/mri_transforms.py

```diff
@@ -1089,6 +1090,8 @@ def build_mri_transforms(
         Number of coils to pad data to.
     scaling_key : str
         Key in sample to scale scalable items in sample. Default: "masked_kspace".
+    scale_percentile : Optional[float]
```

Comment:
I think its: "float, optional". Does this parse correctly?

Reply:
It parses like: Optional[float]. Is that wrong?